### DISCUSSION TITLE
Question about title color

### DISCUSSION THREAD

**Discussion by LeeRenJie** [prabhuignoto/react-chrono#124] [Q&A] [answered]
Is there a way to change the title color? 
![image](https://user-images.githubusercontent.com/67599130/112992907-35975280-919b-11eb-94a9-6cdfd8a0bbb8.png)

I tried to change via themes but still cant.

**Comment by prabhuignoto** [comment, marked as answer]
the `theme` prop has been modified to include a new attribute `titleColor`. You can change color with this new property

**Reply by LeeRenJie** [reply]
Great to hear that! Will give it a go 😄

**Comment by sigmarsson** [comment]
Making the title background color transparent is possible ? 

<img width="214" alt="image" src="https://github.com/user-attachments/assets/e2b08c7f-ffbf-4f00-a08e-92cbd97ce43a">
